index.bs

```diff
@@ -238,7 +239,27 @@ When the `[[track]]` of a {{MediaStreamTrackProcessor}} |processor|
 {{TrackEnded|ends}}, the [=processorClose=] algorithm must be
 executed with |processor| as parameter.
 
-## MediaStreamTrackGenerator interface ## {#track-generator}
+## MediaStreamTrackGenerator ## {#generator}
+A {{MediaStreamTrackGenerator}} allows the creation of a {{WritableStream}}
+that acts as a {{MediaStreamTrack}} source in the
+<a href="https://www.w3.org/TR/mediacapture-streams/#the-model-sources-sinks-constraints-and-settings">
+MediaStream model</a>. Since the model does not expose sources directly
+but through the tracks connected to it, a {{MediaStreamTrackGenerator}}
+is also a track connected to its {{WritableStream}} source. Further tracks
+connected to the same {{WritableStream}} can be created using the
+{{MediaStreamTrack/clone}} method. The {{WritableStream}} source is
+exposed as the {{MediaStreamTrackGenerator/writable}} field of
+{{MediaStreamTrackGenerator}}.
```

Comment:
BTW, what happens in a case like:
1. create MSTG (track A), access writable
2. clone to create track B
3. stop track A

In that case track A would be ended, writable would not be closed, and writing to writable would send frames to track B, correct?

Reply:
Correct.